app/controllers/static_pages_controller.rb

```diff
@@ -2,6 +2,9 @@ class StaticPagesController < ApplicationController
   def home
```

Comment:
```suggestion
  def home; end
```
tương tự cho about, help